.gitignore

```diff
@@ -150,6 +150,7 @@ packages/fxa-react/test/
 
 # fxa-settings
 packages/fxa-settings/fxa-content-server-l10n/
+packages/fxa-settings/public/static
```

Comment:
Not sure where but it feels like we could use a comment somewhere in some file in this PR about this being for l10n/FTL hashing, since webpack does it on our other files.